mobile/native/src/channel_fee.rs

```diff
@@ -79,8 +79,15 @@ impl ChannelFeePaymentSubscriber {
                 funding_tx_fees_msats,
                 "Waiting for outbound capacity on channel to pay jit channel opening fee.",
             );
-            Handle::current()
-                .block_on(self.wait_for_outbound_capacity(channel_id, funding_tx_fees_msats))
+            Handle::current().block_on(async {
+                self.wait_for_outbound_capacity(channel_id, funding_tx_fees_msats)
+                    .await?;
+                // We add another sleep to ensure that the channel has actually been updated after
```

Comment:
it would be nice if we could tag these as `FIXME` or something like that, to find these spots easier, especially when we're bug-hunting in the future!